docs/NOTIFICATIONS.md

```diff
@@ -22,7 +22,34 @@ SPRING_MAIL_PASSWORD="pass"
 
 ### Email template
 Templates ensure consistent and automated communications for various functions. 
-We are using [Thymeleaf](https://www.thymeleaf.org/) as the template engine, the email templates are stored in the `resources/templates/` directory.
+We are using [Thymeleaf](https://www.thymeleaf.org/) as the template engine, which allows us to create dynamic and customizable email content.
+
+#### Template location
+The email template is stored in the `src/main/resources/templates/` directory. It is written in HTML with Thymeleaf syntax, allowing you to include dynamic content using expressions, conditionals, and loops.
+
+#### Editing the template
+ - To edit the template, open the `email-notification.html` file.
+ - Make the necessary changes to the template. You can use Thymeleaf syntax to include dynamic variables, iterate over data collections, or conditionally display content.
+ - Save the changes to the file.
+ - The template is now ready to be used in the email service.
+
+#### Template variables
+The email template uses variables to include dynamic content. These variables are replaced with actual values when the email is sent. The following variables are available in the template:
+ - `recipient`: The name of the recipient.
+ - `frontendUrl`: The URL of the frontend application.
+ - `negotiations`: List of negotiation Ids.
+ - `roleForNegotiation`: List of roles for each negotiation.
+ - `titleForNegotiation`: List of titles for each negotiation.
+ - `notificationsForNegotiation`: List of notifications for each negotiation.
+
+#### Updating Templates at Runtime
+Thymeleaf supports runtime updates to email templates. This allows you to make changes without restarting the application, which is useful for quick updates or tweaks:
+ - **Enable Template Cache Refresh:** By default, Thymeleaf caches templates for performance. To enable runtime updates, you need to configure Thymeleaf to refresh templates automatically. This can typically be done by setting the `spring.thymeleaf.cache` property to `false` in your application configuration file (`application.yml`):
+```yaml
+spring.thymeleaf.cache=false
```

Comment:
maybe set this property for application-dev.yaml? so devs can take advantage of it?